### ISSUE TITLE
Printing layers with combined styles that includes text 

### ISSUE THREAD

**Issue by David0099** [origo-map/origo#1546]
**Describe the bug**
Problem when entering print-mode when a layer that has booth a text style and another style is visible.
For example a point layer styled with both a circle and a text. 
When the print-mode is entered the map freezes. 

**To Reproduce**
Style-example that doesent work. 

```
    "origo-logo": [
        [
            {
                "label": "Origokommuner",
                "circle": {
                    "radius": 10,
                    "stroke": {
                        "color": "rgba(0,0,0,1)",
                        "width": 2.5
                    },
                    "fill": {
                        "color": "rgba(255,255,255,0.9)"
                    }
                }
            },
            {
                "circle": {
                    "radius": 2.5,
                    "stroke": {
                        "color": "rgba(0,0,0,0)",
                        "width": 1
                    },
                    "fill": {
                        "color": "rgba(37,129,196,1)"
                    }
                }
            },
            {
                "text": {
                    "font": "Bold 12px Arial",
                    "textAlign": "center",
                    "offsetX": 10,
                    "offsetY": -10,
                    "text": "My label text",
                    "fill": {
                        "color": "rgba(0,0,0,1.0)"
                    },
                    "stroke": {
                        "color": "rgba(255,255,255,0.7)",
                        "width": 2
                    }
                }
            }
        ]
    ]
```

**Screenshots**
![image](https://user-images.githubusercontent.com/52694843/171369068-2b64a07a-cc1d-492e-b9bf-c10fb992ca08.png)

**Desktop (please complete the following information):**
 - Windows 10
 - Chrome

**Comment by jokd** [comment]
Looks like two bugs related to labels. One is this and the other one is text not being rescaled when leaving the print page.

**Comment by jokd** [comment]
This is hopefully solved in #1550
